srcpkgs/polari/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'polari'
 pkgname=polari
-version=3.36.3
+version=3.38.0
 revision=1
 build_style=meson
 hostmakedepends="pkg-config itstool gobject-introspection gettext"
 makedepends="gjs-devel gspell-devel gtk+3-devel libsecret-devel
- libsoup-gnome-devel telepathy-glib-devel telepathy-logger-devel"
+ libsoup-gnome-devel telepathy-glib-devel telepathy-logger-devel harfbuzz-devel"
```

Comment:
`gtk+3-devel` pulls in `harfbuff-devel` automatically; this addition is superfluous.

Reply:
interesting, it complains about missing harfbuzz when I don't have it in makedepends on my system. I will remove it, thanks